cmd/unset.go

```diff
@@ -35,7 +35,20 @@ func unsetCmd(ctx *cli.Context) error {
 		return errs.NewUsageExitError(msg, ctx)
 	}
 
-	cred, err := setCredential(ctx, args[0], func() *apitypes.CredentialValue {
+	pathexp, _, err := determineCredential(ctx, args[0])
+	if err != nil {
+		return errs.NewErrorExitError("Could not unset credential", err)
+	}
+
+	preamble := fmt.Sprintf("You are about to unset \"%s\". This cannot be undone.", pathexp.String())
+
+	abort := ConfirmDialogue(ctx, nil, &preamble)
```

Comment:
since `ConfirmDialog` retuns an error, we should call it `err` or `abortErr`. Perhaps a better interface would be `bool, err` for `ConfirmDialog` since a CTRL+C and a `Yes` to confirm are different.

You can see a world where we'd want to print out, `Aborted.` or similar.